### ISSUE TITLE
Question about username on the cluster

### ISSUE THREAD

**Issue by JinyuanZhang-Eddie** [carpentries-incubator/hpc-intro#435]
Hi all, I'm new to this lesson and I notice we need a username for the Transfer Your Public Key task. Should I create a new user on the cluster in order to do so? If so, how can I do that without connecting to it for a first time?

**Comment by reid-a** [comment]
Hi Eddie --

The short answer to your question is that the lesson assumes that it's being taught in partnership with cluster administrators, either at a specific site, or on a cloud-based transient cluster specifically set up for instructional purposes. In either case, accounts are created for learners as part of the set-up process.

There is also a deeper question about the practical issues surrounding this. The lesson has been taught successfully several times, and the set-up process is usually not trivial. Some way of streamlining this for lesson sessions, and also some way of making the content useful "off-line", to someone discovering it on the web and trying to work through it on their own, would be useful. We have some ideas, but we don't have any solid answers to this deeper issue.

**Comment by JinyuanZhang-Eddie** [comment]
Thanks for your answer. So as an individual learner, I cannot have access to the server? Or is there a way for me to create a user on the server by myself or is there a universal username that I can use?

**Comment by ocaisa** [comment]
@JinyuanZhang-Eddie That's right, there's no way for us to offer open access to these kinds of resources right now. They would be easily abused and we would have to put a lot of safe guards in place.
